Gems/ROS2/Code/Source/ROS2ModuleInterface.h

```diff
@@ -58,6 +60,7 @@ namespace ROS2
                     ROS2ImuSensorComponent::CreateDescriptor(),
                     ROS2GNSSSensorComponent::CreateDescriptor(),
                     ROS2LidarSensorComponent::CreateDescriptor(),
+                    ROS2Lidar2DSensorComponent::CreateDescriptor(),
```

Comment:
ROS2WheelOdometry seems to be missing.

Reply:
Do you know if these modules are still needed?

```cpp
JointPublisherComponent::CreateDescriptor(),
ManipulatorControllerComponent::CreateDescriptor(),
PidMotorControllerComponent::CreateDescriptor(),
MotorizedJointComponent,
```

Reply:
This is part of features that are in development only and those are needed.
MotorizedJointComponent was refactored and currently lives only in stabilization.

Reply:
Thanks for the info. I'll include those in dev and remove `MotorizedJointComponent`.